PHP 7.1 template generated by swig 3.0.12 doesn't work with zts enabled. This
patch fixes this by using ZEND macros which expand correctly no matter the zts.

This is already fixed in upstream and will be in the new release.

--- swig-3.0.12/Source/Modules/php.cxx	2019-01-10 02:01:31.034906100 +0000
+++ swig-3.0.12/Source/Modules/php.cxx	2019-01-10 01:57:22.571005226 +0000
@@ -370,8 +370,8 @@ public:
     Printf(s_header, "int error_code;\n");
     Printf(s_header, "ZEND_END_MODULE_GLOBALS(%s)\n", module);
     Printf(s_header, "ZEND_DECLARE_MODULE_GLOBALS(%s)\n", module);
-    Printf(s_header, "#define SWIG_ErrorMsg() (%s_globals.error_msg)\n", module);
-    Printf(s_header, "#define SWIG_ErrorCode() (%s_globals.error_code)\n", module);
+    Printf(s_header, "#define SWIG_ErrorMsg() ZEND_MODULE_GLOBALS_ACCESSOR(%s, error_msg)\n", module);
+    Printf(s_header, "#define SWIG_ErrorCode() ZEND_MODULE_GLOBALS_ACCESSOR(%s, error_code)\n", module);
 
     /* The following can't go in Lib/php/phprun.swg as it uses SWIG_ErrorMsg(), etc
      * which has to be dynamically generated as it depends on the module name.
